src/python/WMCore/WorkQueue/Policy/Start/StartPolicyInterface.py

```diff
@@ -237,8 +244,15 @@ def getDatasetLocations(self, datasets):
         """Returns a dictionary with the location of the datasets according to DBS"""
         result = {}
         for dbsUrl in datasets:
-            dbs = self.dbs(dbsUrl)
             for datasetPath in datasets[dbsUrl]:
-                locations = dbs.listDatasetLocation(datasetPath)
+                locations = set()
+                if hasattr(self.rucio, "phedexCompat"):
```

Comment:
I can't figure out why this is returning True... the environment variable is not there, we have a phedex instance and even so it goes into this `if` block!?!?